package-releases/base/cronjob.yaml

```diff
@@ -16,13 +13,9 @@ spec:
   jobTemplate:
     spec:
       template:
-        metadata:
-          labels:
-            app: thoth
-            component: package-releases
         spec:
           containers:
-            - image: "package-releases-job:latest"
+            - image: "package-releases-job"
```

Comment:
Should I keep `latest` tag?

Reply:
No need as it wil use the latest by default, Argo CD make sure what is in the cluster is in sync with versioning from quay :)  @harshad16 can be more verbose here :D

Reply:
latest is not required as @pacospace  suggested.
openshift default looks for `latest` if no tag is defined.
also, imagestreamtag in the package-release-job component would be responsible for latest tag update with new versions along with argocd.